package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "eslint": "eslint *.js benchmark spec",
+    "test-spec-watch": "mocha spec/*.spec.js -R spec --watch",
```

Comment:
You don't really need this. Simply run:

```sh
$ yarn run test-spec --watch
```

and you can just inject the "extra" flags... If you can't remember `--watch`, then it's cool to keep this.